managing-os/templates.md

```diff
@@ -87,6 +87,17 @@ Important Notes
 
        $ sudo yum remove qubes-template-fedora-24
 
+ * Standalone VMs using Template VMs as a basis can be created easily. These
+   VMs receive a *copy* of the operating system and do not get automatically
+   updated when Template VMs are updated--they must be updated individually.
+   
+ * Template VMs can occupy more space on the dom0 filesystem than necessary
+   because they cannot employ automatic TRIM/discard on the root fs. The
+   `qvm-trim-template` command in dom0 is used to recover this unused space.
```

Comment:
Err... are you sure this is true?

IIUC the problem is that the CoW layer doesn't pass trim/discard, and IIUC this layer is not in use for a templatevm's root fs while the template itself has mounting it (r/w).